src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -3959,3 +3992,21 @@ export function showQueryUpdatedToast(message: string) {
         theme: 'light',
     } as any);
 }
+
+export function getDataTypeByDataFilterValues(
```

Comment:
@kalletlak  if i read this correctly, and it's difficult to read, we resolving to type string if any value in dataFilterValues is either undefined or empty string?   First of all, that seems very strange.  What if the values are all numbers?  Then it will resolve to type string?   There should be at least a comment explaining why it behaves this way and some indication in the name of the function of this quirkiness.

Reply:
Karthik and I had a discussion and agreed with you to remove this. We'll giving it datatype instead of determining it in DataFilterValue.